src/hooks/useChainIdBackground.ts

```diff
@@ -1,14 +1,15 @@
 import { SupportedChainId } from 'constants/chains'
-import React from 'react'
+import { useEffect } from 'react'
 import { useActiveWeb3React } from './web3'
 
 export const useChainIdBackground = () => {
   const { chainId } = useActiveWeb3React()
-  React.useEffect(() => {
-    const background = document.querySelector('#background-radial-gradient')
-    if (background === null) {
+  const background = document.querySelector('#background-radial-gradient') as HTMLDivElement | null
```

Comment:
sorry, nit, `document.getElementById('background-radial-gradient')` is faster (not that it matters at all)

Reply:
`getElementById` is not faster than `querySelector` unless maybe you're executing it thousands of times in a loop in IE7 in 2007. Then… maybe 😜